cmd/litestream/main.go

```diff
@@ -440,9 +454,12 @@ type CompactionLevelConfig struct {
 	Interval time.Duration `yaml:"interval"`
 }
 
-// DBConfig represents the configuration for a single database.
+// DBConfig represents the configuration for a single database or directory of databases.
 type DBConfig struct {
 	Path               string         `yaml:"path"`
+	Directory          string         `yaml:"directory"` // Directory to scan for databases
```

Comment:
I'd shorten this down to `Dir` and then `dir` for the config. Not a big deal, just personal preference.